test/unit-test.js

```diff
@@ -0,0 +1,46 @@
+var expect = require('chai').expect;
+var assert = require('chai').assert;
+var should = require('chai').should;
+
+const fs = require('fs');
+const h5 = require('../index')
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+const TEST_FILE = fs.readFileSync('/Users/bcg/Desktop/Tracks/Salute.mp3');
```

Comment:
You probably don't want to hard code in your username and filesystem. This won't work on anyone's machine, but yours.